__tests__/__snapshots__/index.test.tsx.snap

```diff
@@ -0,0 +1,9 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
you can remove this file, snapshot tests are useless, but must be used for specifice use cases